packages/ui/src/components/BenefitCard/BenefitCard.tsx

```diff
@@ -11,16 +12,23 @@ export interface BenefitCardProps extends CenterProps {
 
 export const BenefitCard = ({ icon, title, subtitle, ...props }: BenefitCardProps) => {
   const [hovered, setHovered] = useState(false);
+  const titleSize = useBreakpointValue({ base: 'md', md: 'lg' } as const);
+  const subtitleSize = useBreakpointValue({ base: 'sm', md: 'md' } as const);
+  const iconWrapperSize = useBreakpointValue({ base: '48px', md: '56px' } as const);
+  const iconSize = useBreakpointValue({ base: '24px', md: '32px' } as const);
+  const wrapperPaddingTop = useBreakpointValue({ base: '44px', md: '52px' } as const);
+  const wrapperPaddingBottom = useBreakpointValue({ base: '40px', md: '80px' } as const);
 
   return (
     <Center
       bg="gray.50"
       borderRadius="8px"
-      pt="52px"
-      pb="80px"
+      pt={wrapperPaddingTop}
+      pb={wrapperPaddingBottom}
       px="24px"
       w="100%"
       minHeight="248px"
```

Comment:
🏕️ `minHeight` is needed only when we display many benefits in one row, as we want them to have the same size. When you display only one tile per row you can remove this property and rely solely on paddings.